LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (C) Lassi Kortela, and contributors (2021).
```

Comment:
I probably didn't write most of the current code :) Not sure what should be written here.